src/shim/sdk/DocumentSDK.ts

```diff
@@ -392,7 +392,7 @@ export const advanced = {
         const payload: MT.DocumentUnmanagedEncryptRequest = {
             type: "DOCUMENT_UNMANAGED_ENCRYPT",
             message: {
-                documentData,
+                documentData: documentData.slice(0),
```

Comment:
Why the difference here?

Reply:
No reason, just left over from initial code. Changed.